lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -299,7 +299,7 @@ using namespace Js;
     Var JavascriptOperators::ToNumberInPlace(Var aRight, ScriptContext* scriptContext, JavascriptNumber* result)
```

Comment:
Here too, if this is the spec operation ToNumber, it should throw on BigInt.

Reply:
will remove